<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>401</width>
    <height>822</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Segment Converter</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>2. Segment List</string>
    </property>
   </widget>
   <widget class="QPushButton" name="complete">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>710</y>
      <width>75</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>complete</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>740</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>GMO 운영팀</string>
    </property>
   </widget>
   <widget class="QLabel" name="segmentPrefix_print">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>220</y>
      <width>251</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>440</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>5. Current Segment</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_currentSeg">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>490</y>
      <width>311</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_currentSeg">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>460</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Folder Upload</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_segArc">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>630</y>
      <width>311</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>570</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>6. Segment Archive</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_segArc">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>590</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Folder Upload</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>1. Owner ID</string>
    </property>
   </widget>
   <widget class="QPushButton" name="textLine_owner_enter">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>100</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter</string>
    </property>
   </widget>
   <widget class="QLabel" name="textLine_owner_out">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>130</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>130</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter: </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="textLine_owner">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="as_is_box">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>280</y>
      <width>120</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>3. As-Is</string>
    </property>
    <widget class="QRadioButton" name="as_is_prop">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>90</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Prop</string>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="as_is_evar">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>90</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Evar</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="as_is_entry">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Entry</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="as_is_exit">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>110</y>
       <width>90</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Exit</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="to_be_box">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>280</y>
      <width>120</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>4. To-Be</string>
    </property>
    <widget class="QRadioButton" name="to_be_prop">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>90</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Prop</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="to_be_evar">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>90</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Evar</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="to_be_entry">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>90</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Entry</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="to_be_exit">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>110</y>
       <width>90</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Exit</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_seglist">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>180</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>CSV Upload</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_seglist">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>210</y>
      <width>311</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>760</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>sunkyeong.lee@concentrix.com</string>
    </property>
   </widget>
   <zorder>as_is_box</zorder>
   <zorder>to_be_box</zorder>
   <zorder>label</zorder>
   <zorder>label_4</zorder>
   <zorder>complete</zorder>
   <zorder>label_5</zorder>
   <zorder>segmentPrefix_print</zorder>
   <zorder>label_11</zorder>
   <zorder>textBrowser_currentSeg</zorder>
   <zorder>pushButton_currentSeg</zorder>
   <zorder>textBrowser_segArc</zorder>
   <zorder>label_14</zorder>
   <zorder>pushButton_segArc</zorder>
   <zorder>label_16</zorder>
   <zorder>textLine_owner_enter</zorder>
   <zorder>textLine_owner_out</zorder>
   <zorder>label_17</zorder>
   <zorder>pushButton_seglist</zorder>
   <zorder>textBrowser_seglist</zorder>
   <zorder>textLine_owner</zorder>
   <zorder>label_2</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>401</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
